portfolio/src/main/webapp/index.html

```diff
@@ -4,10 +4,10 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script.js" defer></script>
   </head>
   <body>
-    <div id="content">
```

Comment:
Indentations should be 2 spaces according to the style guide:
https://google.github.io/styleguide/htmlcssguide.html#Indentation